app/src/main/java/com/nervousfish/nervousfish/modules/constants/Constants.java

```diff
@@ -24,6 +24,8 @@
     private static final Logger LOGGER = LoggerFactory.getLogger("Constants");
     private static final String DB_USERDATA_PATH = "accountInformation.json";
     private static final String DB_CONTACTS_PATH = "contacts.json";
+    private static final String DB_DATABASE_PATH = "database.txt";
```

Comment:
Maybe "database" is a bit of a confusing name, we will talk about this in the meeting